tracee/tracee.bpf.c

```diff
@@ -176,6 +180,17 @@ BPF_MAP(_name, BPF_MAP_TYPE_PERF_EVENT_ARRAY, int, __u32, 1024);
 #error Minimal required kernel version is 4.14
 #endif
 
+// Stack Traces are slightly different
+// in that the value is 1 big byte array
+// of the stack addresses
+#define BPF_STACK_TRACE(_name, _max_entries) \
```

Comment:
Can you please move this defintion 10 lines before this point (after `#define BPF_PERF_OUTPUT(_name)`...)